pkg/tenant/db/members.go

```diff
@@ -185,6 +206,8 @@ func UpdateMember(ctx context.Context, member *Member) (err error) {
 		member.Created = member.Modified
 	}
 
+	member.LastActivity = member.Modified
```

Comment:
If `LastActivity` means `LastLogin` then do we only want to update it when the member logs in?